AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -57,7 +57,8 @@ def reduce_compscan_inf(h5,rfi_static_flags=None,chunks=16,return_raw=False,use_
     h5.select(reset='B') # Resets only pol,corrprods,ants
     active_ants = list(set(h5.antlist) & set(find_active_ants(h5, 0.85))) # Only those specified AND active during this compscan
     h5.select(ants=active_ants)
-    
+    h5.antlist = [a.name for a in h5.ants]
```

Comment:
I don't like the piggybacking of data on the `h5` object. Since `h5.bls_lookup` is now finally only accessed within this function, just call it... `bls_lookup`. The same goes for `antlist`. Just recreate it on the outside when needed. Storing this kind of state is what got the script into trouble in the first place.